scripts/qa_crowbarsetup.sh

```diff
@@ -4781,18 +4782,27 @@ function onadmin_crowbar_nodeupgrade
         fi
 
         if [[ $want_nodesupgrade ]]; then
-            # FIXME: crowbarctl command can time out easily
-            # Do not use it until https://bugzilla.novell.com/show_bug.cgi?id=997293 is fixed
-            # safely crowbarctl upgrade services
-            curl -s -X POST -H "$crowbar_api_v2_header" $crowbar_api/api/upgrade/services
+            safely crowbarctl upgrade services
+            wait_for 10 1 "grep -q running $upgrade_progress_file" "services step to start"
+            wait_for 300 5 "! grep -q running $upgrade_progress_file" "services step to finish"
```

Comment:
this way of tracking state is prone to races - e.g. if an upgrade finishes too fast (e.g. under 1 second), you will not see it and error out in the first wait.